res/controllers/Hercules-DJ-Console-4-Mx-scripts.js

```diff
@@ -0,0 +1,622 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+//////////////////////////////////////////////////////////////////////// 
+//=====================================================
+//___ Hercules DJ Console 4-Mx scripts by josepma. ____
```

Comment:
Volume knob: Reworded the wiki entry. 
It only controls Mixxx's main volume knob, opposed to Headphone volume and microphone volume that are hardware controls. 
Soundard volume is controlled from the tray icon application or OS Mixer.

Stop button: changed to cue_gotoandstop as you suggest. Light is active when it is stopped. Ideally it should be lit if it
is stopped and not at the cue pos.

Description only in wiki. Added URL to point to it.

Email address added too.